CHANGELOG/CHANGELOG-1.13.0.md

```diff
@@ -40,6 +42,7 @@
 
 ## Known Issues 
 When using Helm charts to install the PowerMax driver with multiple arrays, the powermax-array-config ConfigMap is incorrectly created, resulting in multiple X_CSI_POWERMAX_ENDPOINT entries. This causes the driver pods to crash with the error "mapping key "X_CSI_POWERMAX_ENDPOINT" already defined". To resolve this issue, you will need to manually edit the ConfigMap powermax-array-config to remove all instances of X_CSI_POWERMAX_ENDPOINT and restart the driver pods (https://github.com/dell/csm/issues/1760).
+After restarting a PowerFlex CSI node pod, any deployment whose pods are scheduled on the same node as the restarted CSI node pod will experience pods stuck indefinitely in the Terminating state when the deployment is restarted via a command such as 'oc rollout restart'. To resolve this issue, upgrade to v1.13.1 (https://github.com/dell/csm/issues/1782).
```

Comment:
Can you rephrase the first sentence? It wasn't clear to me what this meant on the first read through

Reply:
clarified in [commit](https://github.com/dell/csm/pull/1819/commits/a2c19c5592cea2b449828ba137df4bf6e009dd6a)